Task1/Task1.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+#define LENGTH_OF_BIN_NUMBER 256
+
+int main()
+{
+    printf("Enter positive binary integer to convert it to octal integer: ");
+    char binNumber[LENGTH_OF_BIN_NUMBER + 1] = {0}; // + 1 для '\0'
+    int octalNumber[LENGTH_OF_BIN_NUMBER] = {0}; // длина восьмеричного числа точно не больше длины двоичного
```

Comment:
```suggestion
    char octalNumber[LENGTH_OF_BIN_NUMBER / 3 + 2] = {0}; // длина восьмеричного числа точно не больше длины двоичного
```
Мы могли выделить памяти в 3 раза меньше, чем для двоичного представления, а выделили в 4 раза больше (int в 4 раза больше, чем char).
(-0.5)